packages/field-base/test/clear-button-mixin.test.js

```diff
@@ -66,9 +69,30 @@ const runTests = (defineHelper, baseMixin) => {
     it('should focus the input on clear button click', () => {
       const spy = sinon.spy(input, 'focus');
       clearButton.click();
+      mousedown(clearButton);
       expect(spy.calledOnce).to.be.true;
     });
 
+    it('should clear the input value on clear button touch', async () => {
+      touchstart(clearButton);
+      touchend(clearButton);
+      await nextFrame();
+      expect(input.value).to.equal('');
+    });
+
+    it('should not focus the input on clear button touch', () => {
+      const spy = sinon.spy(input, 'focus');
+      touchstart(clearButton);
+      touchend(clearButton);
+      expect(spy.called).to.be.false;
+    });
+
+    it('should keep focus at the input on clear button touch', () => {
```

Comment:
How about we have two tests verifying that the mousedown and touchend events are prevented instead?

```suggestion
    it('should keep focus at the input on clear button touch', () => {
```

Reply:
Would this work ?

```
it('should prevent default action on clear button mousedown', () => {
  const event = mousedown(clearButton);
  expect(event.defaultPrevented).to.be.true;
});
```

Reply:
BTW, `should keep focus at the input on clear button touch` can be removed if we don't prevent default for `touchend` events.

Reply:
> Would this work?

It didn't work for me.

> BTW, `should keep focus at the input on clear button touch` can be removed if we don't prevent default for `touchend` events.

I believe it makes sense to keep it to make sure the behavior is not changed by accident in the future.

Reply:
To clarify further, my argument for the removal was based on the fact that we already test that `focus` isn't called on clearButton during touch events.

Reply:
BTW, to make preventDefault work, you need to make the event cancelable:

```
it('should prevent default on clear button mousedown', () => {
  const event = new CustomEvent('mousedown', { cancelable: true });
  clearButton.dispatchEvent(event);
  expect(event.defaultPrevented).to.be.true;
});
```